libkbfs/folder_branch_ops.go

```diff
@@ -1840,13 +1843,26 @@ func (fbo *folderBranchOps) finalizeMDWriteLocked(ctx context.Context,
 			fbo.log.CDebugf(ctx, "Conflict: %v", err)
 			mergedRev = md.Revision
 
+			if excl == WithEXCL {
```

Comment:
Sure; I think that makes sense! `gorename` makes it super easy to do it so no worries :D